app/views/content/welcome/landing/_youre-not-the-only-one-with-questions.html.erb

```diff
@@ -62,35 +62,35 @@
     </div>
     <div class="questions-and-answers">
       <div class="question-and-answer">
-        <p class="q">
+        <h3 class="q">
```

Comment:
I think the `h3` works well :-) 

If we were going to keep the Welcome guide, I'd suggest we clear up all the classes/figcaptions etc. but I don't think it is worth it now!